packages/shared/routes/onboarding/views/profile-recovery/views/BackupPasswordView.svelte

```diff
@@ -31,7 +31,11 @@
                 if ($onboardingProfile?.setupType === ProfileSetupType.Claimed) {
                     await restoreBackupForShimmerClaimingProfileManager(strongholdPassword)
                 } else {
-                    await restoreBackupFromStrongholdFile(strongholdPassword)
+                    try {
+                        await restoreBackupFromStrongholdFile(strongholdPassword)
+                    } catch {
+                        await restoreBackupFromStrongholdFile(strongholdPassword, true)
+                    }
```

Comment:
Why not move this try/catch statement into `restoreBackupFromStrongholdFile`? You could rid yourself from an argument that is sometimes there, sometimes not and which is already initialized.